go/pinentry/pinentry_x11.go

```diff
@@ -4,6 +4,7 @@ package pinentry
 
 import "os"
 
-func IsRemote() bool {
+func HasWindows() bool {
+	//If there is a DISPLAY then we can spawn a window to it.
 	return len(os.Getenv("DISPLAY")) > 0
```

Comment:
My display is set originally set to `:0` since I am using i3 Window manager in Arch so by simply trying to run `keybase login email@email.com` it errors with `ERROR Failed to get getpin greeting: EOF` so in case the display is `:0` it would be also good to have hasWindow set to `false` here. Otherwise the app won't work unless the user manually unset the DISPLAY variable for that session